### DISCUSSION TITLE
Difference between indic trans and IndicXlit

### DISCUSSION THREAD

**Discussion by Surya291** [AI4Bharat/IndicXlit#13] [Q&A]
Hello, 
Firstly thanks to the team of AI Bharat for open sourcing such models. 

I have been using indic trans for quite some time now and recently came across IndicXlit.  How different is IndicXlit different from indic trans wrt model architecture and performance ?

**Comment by GokulNC** [comment]
Hello Surya, which IndicTrans do you mean?

**Reply by Surya291** [reply]
I have explored both AI Bharat's [IndicTrans ](https://github.com/AI4Bharat/indicTrans) and libindic's [indic-trans](https://github.com/libindic/indic-trans), is there any performance comparision done for AI Bharat's IndicTrans ?

**Reply by GokulNC** [reply]
The one by LibIndic is basically Indic-Transliterate, and the one by AI4Bharat is Indic-Translate.
Both are for different purposes. The former is for [machine transliteration](https://en.wikipedia.org/wiki/Transliteration) and the latter is for [machine translation](https://en.wikipedia.org/wiki/Machine_translation).

The transliteration on LibIndicTrans using a basic perceptron model ([check section 2.2.2 of their paper](https://dl.acm.org/doi/10.1145/2824864.2824872)), and the transliteration on AI4Bharat-Indic-Xlit is using a transformer model ([check the paper for more details](https://arxiv.org/abs/2205.03018)).
